_posts/2020-08-11-dereference-pushdown.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: post
+title:  "Faster Queries on Nested Data"
+author: Pratham Desai
+---
+
+Significant performance improvements have been added in PrestoSQL version 334 
```

Comment:
Make it active voice and present tense.

Presto version 334 adds significant performance improvements for queries...